includes/class-pb-book.php

```diff
@@ -185,6 +185,16 @@ static function getBookStructure( $id = '' ) {
 
 		$q = new \WP_Query();
 
+		/**
+		 * Fetch all pb_export meta values for this book
+		 */
+		global $wpdb;
+		$post_ids_to_export = array();
+		$sql = $wpdb->prepare( "SELECT post_id, meta_value FROM {$wpdb->postmeta} WHERE meta_key = %s", 'pb_export' );
+		foreach ( $wpdb->get_results( $sql, ARRAY_A ) as $val ) {
```

Comment:
This is silly but I think WPCS is dissatisfied because you are putting the `$sql` variable in `$wpdb->get_results()` rather than your `$wpdb->prepare()` function. So you can remove the previous line and do this:

```
foreach ( $wpdb->get_results( $wpdb->prepare( "SELECT post_id, meta_value FROM {$wpdb->postmeta} WHERE meta_key = %s", 'pb_export' ), ARRAY_A ) as $val ) {
```